sst/sstable.go

```diff
@@ -45,6 +46,8 @@ type SSTable struct {
 	//  │maxKeyLength bytes                         │ 4 bytes  │
 	//  ╰───────────────────────────────────────────┴──────────╯
 	data []byte
+	// Varint encoded metadata length
+	metadataSize int
```

Comment:
This is not used, so can be removed.

Reply:
Cleaned up the metadataSize instances. Might have been pre-emptive to think this information maybe necessary. Cleaned it up for now